distribution/sdk/sample-plugins/src/main/java/ddf/sdk/plugin/preingest/DummyPreIngestPlugin.java

```diff
@@ -161,28 +161,4 @@ public DeleteRequest process(DeleteRequest input) throws PluginExecutionExceptio
         return results;
     }
 
-    private List<Metacard> filterOutIds(List<Metacard> cards) {
```

Comment:
✏️ We can probably ignore anything in `/sample*` or `/sdk` since these are example files anyway. I would go ahead and just reset anything within these sample folders if no one disagrees.